### PR TITLE
Update cluster metadata configuration

### PR DESCRIPTION
**Pull request by yux0**
## What does this PR do?
Update cluster metadata configuration with new TCTL command.

## Notes to reviewers
With latest release, we are switching to config multi-cluster via TCTL. The cluster metadata only contains local cluster information.

### CODE REVIEW

**Review by flossypurse — APPROVED** [review]

**Comment by flossypurse** [comment]
If you pushed from a remote, then prettier will fail.
Don't worry about it.

File: docs/server/configuration.md [hunk 1]
```diff
@@ -239,7 +239,7 @@ clusterMetadata:
 - `replicationConsumer` - determines which method to use to consume replication tasks. The type may be either `kafka` or `rpc`.
 - `failoverVersionIncrement` - the increment of each cluster version when failover happens.
 - `masterClusterName` - the master cluster name, only the master cluster can register/update namespace. All clusters can do namespace failover.
-- `clusterInformation` - contains a map of cluster names to `ClusterInformation` definitions. `ClusterInformation` sections consist of:
+- `clusterInformation` - contains the local cluster name to `ClusterInformation` definition. The local cluster name should be consistent with `currentClusterName`. `ClusterInformation` sections consist of:
   - `enabled` - _boolean_ - whether a remote cluster is enabled for replication.
```
Comment (yiminc) [inline docs/server/configuration.md:243]: do we still need this for local cluster?

**Comment by flossypurse** [comment]
@yux0 I think @yiminc comment is for you?
Feel free to merge this when you are ready.

File: docs/server/configuration.md (hunk 1 above)
Comment (yux0) [reply docs/server/configuration.md:243]: Yes

**Pull request merged**